eval/eval-comet.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+set -e
+set -o pipefail
+
+mkdir -p $(dirname "${EVAL_PREFIX}")
+
+sacrebleu -t "$DATASET" -l "$SRC-$TRG" --echo src \
+    | tee "$EVAL_PREFIX.$SRC" \
+    | $TRANSLATOR_CMD \
+    | tee "$EVAL_PREFIX.$TRANSLATOR.$TRG" \
+    ; comet-score --gpus "$GPUS" --quiet -d "$DATASET:$SRC-$TRG" -t "$EVAL_PREFIX.$TRANSLATOR.$TRG" \
```

Comment:
Question: Does comet _require_ GPUs with CUDA? I'm wondering if this is the docker change. I guess this means I won't be able to run this locally.

Reply:
It is not required. You can run it locally on cpu by passing `0` to the --gpus argument, just run outside docker if you are not on Linux.